src/main/java/com/rocketden/main/controller/v1/GameController.java

```diff
@@ -0,0 +1,38 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.dto.room.GetRoomRequest;
+import com.rocketden.main.dto.room.RoomDto;
+import com.rocketden.main.service.RoomService;
+import com.rocketden.main.dto.game.StartGameRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameController extends BaseRestController {
+    
+    private final RoomService service;
+
+    @Autowired
+    public GameController(RoomService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/start")
```

Comment:
What are your thoughts on making this something like `/rooms/{roomId}/start`? Just to match with the new (?) convention we started (idk if we actually all agreed on it so we can also talk about it during standup)

Reply:
Sure, I think putting the room ID in it would be good.